content/plugin/sdkv2/sdkv2-intro.mdx

```diff
@@ -5,9 +5,9 @@ description: Learn about version 2 of the Terraform Plugin SDK.
 
 # Terraform Plugin SDKv2
 
-Terraform Plugin SDKv2 is an established way to develop Terraform Plugins. It is the precursor to the [Terraform Plugin Framework](/plugin/framework), which is still in early development.
+Terraform Plugin SDKv2 is an established way to develop Terraform Plugins on [Protocol Version 5](/plugin/how-terraform-works#terraform-plugin-protocol).
```

Comment:
Verifying a few links, cc @laurapacilio 

This looks good.
https://terraform-website-pboetpxng-hashicorp.vercel.app/plugin/how-terraform-works#terraform-plugin-protocol